app/about/people.html

```diff
@@ -33,6 +34,21 @@
   </ul>
 </section>
 
+<section id="managers">
+  <h2>Our Managers</h2>
+  <ul class="people-list officers-list">
+  {% for officer in site.data.managers %}
+    <li class="person officer" data-img-url="{{ officer.lname | append: ' ' | append: officer.fname | replace: ' ','-' | downcase | append: '.jpg' }}">
+      <img src="/img/people/{{officer.fname | append: '_' | append: officer.lname | append: '.jpg' }}" onerror="this.src='/img/people/generic.jpg'">
```

Comment:
are these as big as the officer photos? IMO they should be about 1/2 or 2/3 the size.